Assets/Scripts/Fade/Fade.cs

```diff
@@ -0,0 +1,49 @@
+using UnityEngine;
+using UnityEngine.UI;
+using LekAanDek.Variables;
+
+namespace LekAanDek.UI.FadeOut
```

Comment:
No need to put it in FadeOut namespace LekAanDek.UI is fine.

Reply:
And please change the folder to UI then. This is to reflect the folder structure.